installing_gitops/docinfo.xml

```diff
@@ -0,0 +1,11 @@
+<title>Installing GitOps</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Installing the Openshift GitOps Operator and logging in to the Argo CD instance</subtitle>
```

Comment:
```suggestion
<subtitle>Installing the OpenShift GitOps Operator and logging in to the Argo CD instance</subtitle>
```
See the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#openshift-cli